frontend/src/js/map.js

```diff
@@ -902,26 +904,34 @@ function getRange(data){
     var circles = _.matches({type: "tract-circle" });
     var circleFilter = _.filter(data, circles);
 
+    // If no tracts in view, return blank array
     if ( circleFilter.length === 0 ){
-        return [];
+        return valArray = [];
     }
+
     // Get the min and max
     var max = _.max(data, function(circleObj){ return circleObj._mRadius; });
     var min = _.min(data, function(circleObj){ return circleObj._mRadius; });
 
     // Determine the midpoint LAR counts for min / max
-    var multiple = (max.volume - min.volume)/3;
-    var drawNewArray = [(min.volume + multiple), (min.volume + (multiple*2))];
+    var multiple = (max.volume - min.volume)/2;
+    var drawNewArray = [(min.volume + multiple)];
 
     var keyCircles, keyCirclesFilter, valArray;
+    
+    // If the maximum volume is less than 2, only return the single max point.
+    if ( max.volume < 2 && max.volume > 0){
+        return valArray = [max];
+    // if max volume for all tracts for the lender is zero, return empty array
+    } else if ( max.volume === 0 ){
+        return valArray = [];
+    }
```

Comment:
Have to keep these checks at the top to exit out of the method to avoid running algorithm that populates valArray.